pymatviz/utils/testing.py

```diff
@@ -0,0 +1,8 @@
+"""Testing related utils."""
+
+from __future__ import annotations
+
+from pymatviz.utils import ROOT
+
+
+TEST_FILES: str = f"{ROOT}/tests/files"
```

Comment:
This might seem a bit overkill, but I kind of want to reserve this for more test functions like #208